### DISCUSSION TITLE
Any way to hide InlineKeyboardMarkup after getting a button pressed?

### DISCUSSION THREAD

**Discussion by dncastilho** [eternnoir/pyTelegramBotAPI#1932] [Q&A]
I have a couple buttons with callbacks and I was wondering - anyway to hide then after the user picks an option?

**Comment by artempas** [comment]
Use edit_message_text method and pass empty reply_markup there
https://pytba.readthedocs.io/en/latest/sync_version/index.html#telebot.TeleBot.edit_message_text
